code/controllers/subsystem/tickets/tickets.dm

```diff
@@ -94,6 +94,8 @@ SUBSYSTEM_DEF(tickets)
 
 	//Inform the user that they have opened a ticket
 	to_chat(C, "<span class='[span_class]'>You have opened [ticket_name] number #[(getTicketCounter() - 1)]! Please be patient and we will help you soon!</span>")
+	var/ticket_open_sound = sound('sound/effects/adminticketopen.ogg')
```

Comment:
Keep the old bwoink sound

Reply:
Can the old bwoink sound be kept. Thanks

Reply:
I assume you commented on the wrong line because this is the ticket opening sound, not the PM reply sound (bwoink) which isn't the new bwoink sound. The old bwoink sound has been kept.